delivery_postlogistics_dangerous_goods/postlogistics/web_service.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+import re
+
+from odoo import _, exceptions
+
+from odoo.addons.delivery_postlogistics.postlogistics import web_service
+
+UNNUMBER_REGEX = re.compile("^[0-9]{1,4}$")
+
+
+class PostlogisticsWebServiceDangerousGoods(web_service.PostlogisticsWebService):
```

Comment:
Not related to this PR, but as the inheritance of this web service class is quite wobbly, I don't know if we should change it to a Component one day.